### ISSUE TITLE
Problem with the central button 

### ISSUE THREAD

**Issue by SiXo1** [reezliabi/aerduino-valorant-cheat#5]
I am using "doge_mouse" because it is the only one that moves the cursor and uses the additional buttons of my mouse, (Pictek PC257A) the click of the central button (wheel) works but when I try to scroll down or up it does not detect it, can you help me please?

**Comment by reezliabi** [comment]
You need to modify the doge_mouse source (.ino file). I have Logitech G502 Hero and all of them are working fine 👍

**Comment by SiXo1** [comment]
> You need to modify the doge_mouse source (.ino file). I have Logitech G502 Hero and all of them are working fine 👍

yes but in which parts should it be modified? apparently everything is fine but with some mice it does not function correctly, with a more basic one the pointer does not even move.

**Comment by SiXo1** [comment]
> > You need to modify the doge_mouse source (.ino file). I have Logitech G502 Hero and all of them are working fine 👍
> 
> yes but in which parts should it be modified? apparently everything is fine but with some mice it does not function correctly, with a more basic one the pointer does not even move.

Now the problem I have is that the script does not open, it opens but closes immediately.

**Comment by Yutooop** [comment]
its because this source is trash!
